runtime/local/src/lib.rs

```diff
@@ -846,6 +852,24 @@ pub type Executive = frame_executive::Executive<
     AllPalletsWithSystem,
 >;
 
+#[derive(Default)]
+pub struct DappsStakingChainExtension;
+
+impl RegisteredChainExtension<Runtime> for DappsStakingChainExtension {
+    const ID: u16 = 00;
+}
```

Comment:
Not related to review in particular - but where do we stand with chain-extension registry? Will we be using that?

Reply:
After a discussion with Alex from Parity. All your custom pallet chain-extension should be encapsulated under id = 0
For the other pallet - that will be used in different runtimes (xcm - assets - balances - rmrk) it should be registered in the chain-extension registry repo:
https://github.com/paritytech/chainextension-registry